config/application.yml.default

```diff
@@ -411,6 +411,7 @@ development:
   identity_pki_local_dev: true
   in_person_proofing_enabled: true
   in_person_proofing_enforce_tmx: true
+  in_person_proofing_opt_in_enabled: true
```

Comment:
I did this taking great care to ensure that it wouldn't inadvertently get enabled in prod or other environments.

But it is now occurring to me that (1) it _is_ enabled in prod and other environments now, and (2) `application.yml.default` is not deployed to prod. 😆

I've only been here coming on 2 years; I'm still learning! 😉

Reply:
Yeah, though to that end, I'm partly asking _because_ I suspect the tests may not pass with this enabled, and _because_ we probably should have made this the default before going live in production, for the precise reason that we don't have test coverage that mimics a configuration that production users will experience. I could be wrong though!

The only other thing I'd want to be careful about is if we explicitly didn't want this enabled in certain deployed environments, because "production" as defined in this file includes sandbox environments. If we don't want the feature enabled anywhere, we'll want to set the configuration override in those environments before changing the default in `main`.

Reply:
So actually this breaks a ton of stuff because the how_to_verify controller only displays if IPP is enabled. So to fully enable opt_in I need to also enable IPP generally (which we should do, but IDK if this PR is the right place, doubtful). 
I'm unsure what the best next step is. 
Options:
1. Just enable Opt In in Dev for the time being and open a separate PR later that enables IPP and Opt In as the defaults. 
2. Enable Opt In everywhere as a default, but set the value to 'false' in specs that expect it to be false. Probably not a great idea as this is making a good amount of stuff to keep track of and clean up later. 
3. Enable IPP and Opt In in this PR and clean up a ton of specs and make this really small PR into an absolutely nightmarish one.

Reply:
@aduth I think I'm going to go with my described option 1 above. Let me know if you have any questions or concerns

Reply:
I'd be okay with doing this incrementally. Though as far as enabling IPP generally, is that something we can break up, or is your third option inevitable?

Reply:
Re: step 3 being inevitable: I'm not totally sure. I think part of the issue is the way we use helpers for our feature specs (by feature spec I mean anything that uses Capybara to step each page in the app) and the nature of the change (opt in being a new page, and being dependent on IPP). For feature specs that rely on doc_auth_helper, I think you kinda have to do them all at once, unless you want to stop using doc_auth_helper which is a path to more work and greater confusion.

For non-feature specs, I think you can probably do it incrementally without much trouble? I think you would have to start by disabling Opt In and IPP in specs that initially assume those values to be false, and this could give you the isolation you need to start in on specific specs.

I said "I think" a lot in my response and this should indicate that I'm still reasoning this out.